images/win/scripts/Tests/Browsers.Tests.ps1

```diff
@@ -0,0 +1,160 @@
+Describe "Chrome" {
+    Context "WebDriver" {
+        It "ChromeWebDriver environment variable exists" {
+            $env:ChromeWebDriver | Should -Not -BeNullOrEmpty
+        }
+
+        It "ChromeWebDriver should be equal C:\SeleniumWebDrivers\ChromeDriver" {
+            $env:ChromeWebDriver | Should -BeExactly "C:\SeleniumWebDrivers\ChromeDriver"
+        }
+
+        It "ChromeWebDriver path exists" {
+            $env:ChromeWebDriver | Should -Exist
+        }
+
+        It "chromedriver.exe exists" {
+            "$env:ChromeWebDriver\chromedriver.exe" | Should -Exist

```

Comment:
Let's also add one more test `"$env:ChromeWebDriver\chromedriver.exe --version" | Should -ReturnZeroExitCode`

Reply:
The same for other tools